bin/run_lint

```diff
@@ -267,7 +267,7 @@ rv=0
 #                                           pylint |    cppcheck
 export PYTHONPATH=$PYTHONPATH:$top_dir/examples/common/python
 lint_module tests                    "$SINCE" 1  0   0  0 || rv=1
-lint_module examples/enclave_manager "$SINCE" 1  0   0  0 || rv=1
+lint_module enclave_manager          "$SINCE" 1  0   0  0 || rv=1
 lint_module examples/common          "$SINCE" 1  0   0  0 || rv=1
```

Comment:
Shouldn't we change example/common -> common/python. May be in a separate PR

Reply:
Yes, will do it in separate PR